src/dbally/views/sqlalchemy_base.py

```diff
@@ -21,15 +21,19 @@ def __init__(self, sqlalchemy_engine: sqlalchemy.engine.Engine) -> None:
 
     @abc.abstractmethod
     def get_select(self) -> sqlalchemy.Select:
-        """
-        Creates the initial SqlAlchemy select object, which will be used to build the query.
+        r"""
+        Creates the initial
+        [SqlAlchemy select object
+        ](https://docs.sqlalchemy.org/en/20/core/selectable.html#sqlalchemy.sql.expression.Select)
+        which will be used to build the query.
         """
 
     async def apply_filters(self, filters: IQLQuery) -> None:
         """
-        Applies the chosen filters to the view.
+        Applies the chosen filters to the view, using the SQL `where` statement.
```

Comment:
" using the SQL `where` statement" - seems like an implementation detail

Reply:
Yes you are right, I somehow wanted to give a little bit more details to the users. But why should they care about it.